src/frontend/screens/Settings/ProjectSettings/index.tsx

```diff
@@ -1,112 +1,209 @@
+      }
+      onPress={() =>
+        navigate(isSolo ? 'InviteCollaborators' : 'ProjectSettings')
+      }
+    />
+  );
+};
 
-  const remoteArchiveOn = remoteArchives && remoteArchives.length > 0;
+const CollaboratorsCard = ({role}: {role: 'coordinator' | 'participant'}) => {
+  const {formatMessage} = useIntl();
+  const {navigate} = useNavigationFromRoot();
 
-  const {data: projects} = useManyProjects();
+  return (
+    <SettingsCardRow
+      icon={<ProjectParticipantIcon width={24} height={24} />}
+      title={formatMessage(m.projectCollaborators)}
+      subtitle={
+        role === 'coordinator'
+          ? formatMessage(m.coordinator)
+          : formatMessage(m.participant)
+      }
+      buttonText={formatMessage(m.viewTeam)}
+      onPress={() => navigate('YourTeam')}
+    />
+  );
+};
 
-  const MenuItems: MenuListItemType[] = [
-    {
-      onPress: () => {
-        navigation.navigate('DeviceNameDisplay');
-      },
-      disabled: remoteArchives === undefined,
-      primaryText: formatMessage(m.deviceName),
-      testID: 'PROJECT.device-name-list-item',
-    },
-    {
-      onPress: () => {
-        navigation.navigate('Config');
-      },
-      primaryText: formatMessage(m.config),
-      testID: 'settingsConfigButton',
-    },
-    {
-      onPress: () => {
-        navigation.navigate('YourTeam');
-      },
-      primaryText: formatMessage(m.yourTeam),
-      testID: 'MAIN.team-list-item',
-    },
-    ...(projects && projects.length > 1
-      ? [
-          {
-            onPress: () => {
-              navigation.navigate(
-                remoteArchiveOn ? 'RemoteArchiveOn' : 'RemoteArchiveOff',
-              );
-            },
-            primaryText: formatMessage(m.RemoteArchive),
-            secondaryText: isRefetching ? (
-              <UIActivityIndicator size={25} />
-            ) : remoteArchiveOn ? (
-              formatMessage(m.remoteArchiveOn)
-            ) : (
-              formatMessage(m.remoteArchiveOff)
-            ),
-          },
-        ]
-      : []),
-    {
-      onPress: () => {
-        navigation.navigate('MediaSyncSettings');
-      },
-      primaryText: formatMessage(m.mediaSyncSettings),
-      testID: 'MAIN.sync-list-item',
-    },
-  ];
+const ConfigCard = () => {
+  const {formatMessage} = useIntl();
+  const {data} = useProjectSettings();
+  const {navigate} = useNavigationFromRoot();
 
-  return <FullScreenMenuList data={MenuItems} />;
+  return (
+    <SettingsCardRow
+      icon={<ProjectCategoriesIcon width={24} height={24} />}
+      title={formatMessage(m.configTitle)}
+      subtitle={data.configMetadata?.name}
+      buttonText={formatMessage(m.updateCategories)}
+      onPress={() => navigate('Config')}
+    />
+  );
 };
 
+const SettingsCardRow = ({
+  icon,
+  title,
+  subtitle,
+  buttonText,
+  onPress,
+}: {
+  icon: React.ReactNode;
+  title: string;
+  subtitle?: string;
+  buttonText?: string;
+  onPress?: () => void;
+}) => (
```

Comment:
the type should be

```ts
type SettingCardRowProps =
  | {
      icon: React.ReactNode;
      title: string;
      subtitle?: string;
    }
  | {
      icon: React.ReactNode;
      title: string;
      subtitle?: string;
      buttonText: string;
      onPress: () => void;
    };
```

so that you onPress and Button have to exist at the same time